database/schema.sql

```diff
@@ -167,6 +167,35 @@ CREATE TABLE updating_email_lists_additions (
   PRIMARY KEY (listname, email)
 );
 
+-- ROTATION TABLES
+
+CREATE TABLE rotation_prefrosh (
+  prefrosh_id INTEGER NOT NULL AUTO_INCREMENT,
+  first_name VARCHAR(255) NOT NULL,
+  last_name VARCHAR(255) NOT NULL,
+  preferred_name VARCHAR(255),
+  image LONGBLOB,
+  bucket ENUM('-2', '-1', '000', '0', '0.5', '1', '1.5', '2', '3') NOT NULL DEFAULT '0',
```

Comment:
This works, but is pretty messy since all the buckets are hard coded here (and in the names of the votes_* columns), and it would be a lot of work if any changes were made to the bucket system. I think a better way of doing this would be to have a "buckets" table:
```
CREATE TABLE buckets (
  bucket_id INTEGER NOT NULL AUTO_INCREMENT,
  bucket_name VARCHAR(10),
  PRIMARY KEY (bucket_id)
);
```
And a "votes" table:
```
CREATE TABLE votes (
  prefrosh_id INTEGER NOT NULL,
  bucket_id INTEGER NOT NULL,
  vote_count INTEGER NOT NULL DEFAULT 0,
  PRIMARY KEY (prefrosh_id, bucket_id)
);
```

Give the time constraint I understand if you don't want to change this right now.